runtime/sema/errors.go

```diff
@@ -3036,7 +3036,7 @@ func (*InvalidAccessError) IsUserError() {}
 
 func (e *InvalidAccessError) Error() string {
 	return fmt.Sprintf(
-		"cannot access `%s`: %s has %s access",
+		"cannot access `%s`: %s requires `%s` authorization",
```

Comment:
should the field `RestrictingAccess` be also changed accordingly? e.g. `RestrictingAccess` -> `RequiredAccess`

Reply:
This was a temporary improvement. @dsainati1 has now improved this and other error messages for entitlements in #2730